src/app/campaign-details/campaign-details.component.ts

```diff
@@ -32,13 +30,13 @@ export class CampaignDetailsComponent implements OnInit {
   }
 
   ngOnInit() {
-    this.campaign = this.state.get(CAMPAIGN_KEY, undefined);
```

Comment:
@NoelLH  could we not have `campaign-${this.campaignId}` as a shared var? Similarly for `fund-for-${this.campaignId}` ?

Reply:
Done and added typing to keys @tbg-jasonfung